src/Raven.Server/Documents/PeriodicExport/PeriodicExportRunner.cs

```diff
@@ -406,7 +406,7 @@ private void WriteStatus()
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out context))
             using (var tx = context.OpenWriteTransaction())
             {
-                _database.DocumentsStorage.SetDatabasePeriodicExportStatus(context, _status);
+                _database.ConfigurationStorage.PeriodicBackupStorage.SetDatabasePeriodicBackupStatus(context, _status);
```

Comment:
We need to change how we persist the information here, instead of storing it in the documents storage, we want to store that in the configuration storage.
That avoids locking the storage for documents, and make it simpler for us.
It also means that for DR, we can just look at the docs storage and not have to figure out what configuration entries mean.